packages/clay-tooltip/src/TooltipProvider.tsx

```diff
@@ -139,14 +139,15 @@ const TooltipProvider: React.FunctionComponent<{
 	const titleNodeRef = React.useRef<HTMLElement | null>(null);
 	const tooltipRef = React.useRef<HTMLElement | null>(null);
 
-	const handleHide = ({target}: any) => {
+	const handleHide = useCallback(({target}: any) => {
 		if (!titleNodeRef.current) {
 			return;
 		}
 
 		const dataTitle = titleNodeRef.current.getAttribute('data-title');
 
 		if (dataTitle) {
+			document.removeEventListener('keyup', handleEsc, true);
```

Comment:
I'm interested why you're providing `true` as the 3rd argument here, as per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) it's an `options` argument. Does it mean that the 3rd argument is implicitly `capture`:
> `capture`: A Boolean which indicates that events of this type will be dispatched to the registered listener before being dispatched to any EventTarget beneath it in the DOM tree.

Reply:
@kresimir-coko yeah, essentially I want to be able to capture the event as soon as possible to stop it, so I add `true`, otherwise, I would be picking up on Bubbling which would be the last phase.

The idea is to close the tooltip with Esc but there may be problems when it is in a Modal or DropDown for example, that Esc closes both, so I need to get the event first and cancel and close the Tooltip.

Reply:
👍  thanks for the explanation